src/reducers/systemIntakeReducer.ts

```diff
@@ -140,6 +141,14 @@ function systemIntakeReducer(
       };
     case archiveSystemIntake.SUCCESS:
       return initialState;
+    case issueLifecycleIdForSystemIntake.SUCCESS:
+      return {
+        ...state,
+        systemIntake: {
+          ...state.systemIntake,
+          ...prepareSystemIntakeForApp(action.payload)
```

Comment:
> What is this `response`? I'm seeing a `POST` and `204 No Content`. I'm not sure what I should be seeing.
> 
> If what i'm seeing is correct, we don't need to pass anything on success to the reducer.

If there's `No Content` in that response, `action.payload` will have nothing and that mess with the data in the store.